README.md

```diff
@@ -134,8 +136,24 @@ Learn more about ONNX
 - [ONNX website](http://onnx.ai/)
 - [ONNX on GitHub](https://github.com/onnx/onnx)
 
-### Operators supported
-The [list](./docs/operators.md) of ONNX operators supported by each of the 3 available builtin backends (cpu, wasm, and webgl).
+### Compatibility
+#### Desktop Platforms
+| OS/Browser | Chrome | Edge | FireFox | Safari | Opera | Electron | Node.js |
+|:----------:|:------:|:----:|:-------:|:------:|:-----:|:-----:|:-----:|
+| Windows 10 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| macOS | :heavy_check_mark: | - | :heavy_check_mark: | Coming soon | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Ubuntu LTS 18.04 | :heavy_check_mark: | - | :heavy_check_mark: | - | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+
+#### Mobile Platforms
+| OS/Browser | Chrome | Edge | FireFox | Safari | Opera |
+|:----------:|:------:|:----:|:-------:|:------:|:-----:|
+| iOS | Coming soon | - | Coming soon | Coming soon | Coming soon |
+| Android | :heavy_check_mark: | :heavy_check_mark: | Coming soon | - | :heavy_check_mark: |
+
+### Operators
+ONNX.js currently supports most operators in [ai.onnx](https://github.com/onnx/onnx/blob/master/docs/Operators.md) operator set v7 (opset v7). See [operators.md](./docs/operators.md) for a complete, detailed list of which ONNX operators are supported by the 3 available builtin backends (cpu, wasm, and webgl).
```

Comment:
same as the comment in ./docs/operators.md - 'master' vs 'rel-1.2.3' in the URL...